mmv1/third_party/terraform/services/compute/resource_compute_instance_from_machine_image_test.go.tmpl

```diff
@@ -1031,6 +1156,63 @@ resource "google_compute_instance_from_machine_image" "foobar" {
 `, projectID, projectID, org, billingId, instance, instance, newInstance)
 }
 
+func testAccComputeInstanceFromMachineImage_encryptionKeyKMS(context map[string]interface{}) string {
+	return fmt.Sprintf(`
+resource "google_compute_instance_from_machine_image" "foobar" {
+		provider = google-beta
+		name = "%s"
+		zone = "us-central1-a"
+		source_machine_image = google_compute_machine_image.foobar.self_link
+		source_machine_image_encryption_key {
+			kms_key_name = "%s"
+			kms_key_service_account = "%s"
+		}
+	}
+`, context["instance_name"].(string), context["kms_key_name"].(string), context["kms_key_service_account"].(string))
+}
+
+func testAccComputeInstanceFromMachineImage_encryptionKeyRaw(context map[string]interface{}) string {
+	return fmt.Sprintf(`
+resource "google_compute_instance_from_machine_image" "foobar" {
+		provider = google-beta
+		name = "%s"
+		zone = "us-central1-a"
+		source_machine_image = google_compute_machine_image.foobar.self_link
+		source_machine_image_encryption_key {
+			raw_key = "%s"
+		}
+	}
+`, context["instance_name"].(string), context["raw_key"].(string))
+}
+
+func testAccComputeInstanceFromMachineImage_encryptionKeySha(context map[string]interface{}) string {
+	return fmt.Sprintf(`
+resource "google_compute_instance_from_machine_image" "foobar" {
+		provider = google-beta
+		name = "%s"
+		zone = "us-central1-a"
+		source_machine_image = google_compute_machine_image.foobar.self_link
+		source_machine_image_encryption_key {
+			sha256 = "%s"
```

Comment:
According to the docs sha256 is output only: https://cloud.google.com/compute/docs/reference/rest/v1/instances

So how can this possibly work?

Reply:
Still present